### PR TITLE
[HELIX-60] CORS Enabled Dashboard for Apache Helix

### PR DESCRIPTION
**Pull request by AyolaJayamaha**
Resolved Cross Domain Issue in helix REST API

### CODE REVIEW

File: helix-hawtio-plugin/pom.xml [hunk 1]
```diff
@@ -0,0 +1,295 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```
Comment (kanakb) [inline helix-hawtio-plugin/pom.xml]: This file is missing the Apache license header (see other pom.xml files).

File: helix-hawtio-plugin/src/main/java/io/hawt/helixplugin/PluginContextListener.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
+package io.hawt.helixplugin;
+
```
Comment (kanakb) [inline helix-hawtio-plugin/src/main/java/io/hawt/helixplugin/PluginContextListener.java]: Please add Apache license header to all new files.

**Comment by kanakb** [comment]
This generally looks good, but the following are necessary before it can be merged:
- All new files need an Apache license header.
- This code should be rebased off of the latest Helix master so that the gray icon above becomes green.
- The 90+ commits should be squashed into a single commit that describes this work.

**Comment by AyolaJayamaha** [comment]
Hi Kanak,
Thanks for the respond. I'll do the needful and get back to you.
Regards.

**Pull request closed without merging**